src/main/scala/firrtl/backends/verilog/VerilogEmitter.scala

```diff
@@ -458,6 +458,8 @@ class VerilogEmitter extends SeqTransform with Emitter {
   private[firrtl] class EmissionOptions(annotations: AnnotationSeq) {
     // Private so that we can present an immutable API
     private val memoryEmissionOption = new EmissionOptionMap[MemoryEmissionOption](MemoryEmissionOptionDefault)
+    private val memoryInitEmissionOption =
+      new EmissionOptionMap[MemoryInitEmissionOption](MemoryInitBlockEmissionOption)
```

Comment:
Is the intent for this to be more of a global option or controllable _per memory_? `EmissionOptionMaps` have machinery for handling different options at a granularity of per component (in this case per memory), but I think that's overkill for what you're doing.

Since `getMemoryInitEmissionOption` doesn't take a `ReferenceTarget` as an option, I think we can simplify this a bit and reduce the public API exposure.

Reply:
I also think this should be a global option. (i.e. a NoTargetAnnotation)